libs/components/molecules/src/lib/interactive-svg/interactive-svg.component.ts

```diff
@@ -139,16 +141,20 @@ export class InteractiveSvgComponent implements OnDestroy {
    * @param event Mouse event
    */
   private onCrosswalkHover(event: MouseEvent): void {
-    const id = this.decodeId(this.getId(event));
-    if (id) {
-      this.nodeHover.emit(id);
+    const id = this.getId(event);
+    let label = '';
+
+    const mapEntry = this.mapping?.find((item) => item['node_name'] === id); //search mapping data for node entry
+    if (mapEntry) {
+      label = mapEntry['label'];
       this.nodeHoverData$.next({
-        node: id,
+        node: label,
         origin: {
           x: event.clientX,
           y: event.clientY,
         },
       });
+      this.nodeHover.emit(mapEntry); //emits node entry and shows node name in tooltip if matching entry is found
```

Comment:
This comment doesn't make sense since it is the above emit that shows the toolip